auto_nag/bzcleaner.py

```diff
@@ -127,6 +130,17 @@ def all_include_fields(self):
     def get_max_ni(self):
         return -1
 
+    def get_max_actions(self):
+        return -1
+
+    def exclude_no_action_bugs(self):
+        """
+        If True, then remove bugs that have no actions from the email (e.g.,
+        needingo got ignored due to exceeding the limit). This is apply only
+        when using the `add_prioritized_action()` method.
+        """
```

Comment:
Could you also add something like `E.g. could be useful if we want to list all actions the tool would do if it had no limits`.

Reply:
Done in b08f98a611e130b63b4f180fc524b135735d88c7